xml/System.Security.Cryptography.X509Certificates/CertificateRequest.xml

```diff
@@ -813,9 +814,44 @@ public static string PemEncodeSigningRequest(CertificateRequest request, PkcsSig
         <param name="signatureGenerator">The signature generator with which to sign the request.</param>
         <summary>Creates an ASN.1 DER-encoded PKCS#10 CertificationRequest representing the current state of the current object using the provided signature generator.</summary>
         <returns>A DER-encoded certificate signing request.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+When submitting a certificate signing request via a web browser, or other graphical or textual
+              interface, the input is frequently expected to be in the PEM (Privacy Enhanced Mail) format,
+              instead of the DER binary format.
+
+          ]]></format>
+        </remarks>
         <exception cref="T:System.ArgumentNullException">
           <paramref name="signatureGenerator" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.InvalidOperationException">
+          <see cref="P:System.Security.Cryptography.X509Certificates.CertificateRequest.OtherRequestAttributes" /> contains a <see langword="null" /> value.
+
+-or-
+
+<see cref="P:System.Security.Cryptography.X509Certificates.CertificateRequest.OtherRequestAttributes" /> contains an entry with a <see langword="null" /><see cref="P:System.Security.Cryptography.AsnEncodedData.Oid" /> value.
```

Comment:
```suggestion
<see cref="P:System.Security.Cryptography.X509Certificates.CertificateRequest.OtherRequestAttributes" /> contains an entry with a <see langword="null" /> <see cref="P:System.Security.Cryptography.AsnEncodedData.Oid" /> value.
```

I assume that a space is required here, otherwise I'd expect to see something like

[OtherRequestAttributes](#OtherRequestAttributes) contains an entry with a **null**[AsnEncodedData.Oid](https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.asnencodeddata.oid?view=net-6.0) property.

(lacking a space between the words).

There were a couple of spaces, and a newline, in the source code:  https://github.com/dotnet/runtime/blob/e71a9583b4d6c9bd97edd87cda7f98f232f63530/src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/CertificateRequest.cs#L301-L304.

If the generated pages don't have an issue, then I'll try to remember that these two things abutting is OK :smile:.

Reply:
Yep, it's missing a space in the output.

![image](https://user-images.githubusercontent.com/10642668/183533181-5f7defce-6bb9-4d93-ab8f-4b781945f17d.png)


Both here, and two lines down.